dev-docs/modules/userId.md

```diff
@@ -853,6 +853,36 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### NextRoll ID
+
+NextRoll is an industry-leading marketing technology and data stack that fuels growth for businesses of all kinds. Our technology powers two multi-million dollar high-growth businesses: AdRoll and RollWorks. The NextRoll ID helps publishers access unique demand in cookieless environments.
+
```

Comment:
this is just an opinion, but this doesn't tell me anything about how your sub adapter actually works, this text seems more appropriate on your corp website.  Other adapter creators have described the inputs and outputs their module receives, or the types of signal (1pc, email, etc) their module uses to resolve their id.

Reply:
I'm working with my team on this description.

In the meantime, I've addressed the other requested changes.